ush/templates/FV3LAM_wflow.xml

```diff
@@ -439,6 +442,103 @@ MODULES_RUN_TASK_FP script.
 ************************************************************************
 ************************************************************************
 -->
+
+{% if run_task_get_obs_ccpa %}
+  <task name="&GET_OBS_CCPA_TN;" maxtries="2">
+
+    &RSRV_HPSS;
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&GET_OBS_CCPA_TN;" "&JOBSDIR;/JREGIONAL_GET_OBS_CCPA"</command>
```

Comment:
This is where I was thinking one GET_OBS_TN would be. But maybe that doesn't work?